console/console-init/ui/src/components/MetadataListTable/TableRow.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React, { useState } from "react";
+import {
+  DataListItem,
+  DataListItemRow,
+  DataListCell,
+  DataListToggle,
+  DataListContent,
+  DataListItemCells
+} from "@patternfly/react-core";
+import classNames from "classnames";
+import { StyleSheet } from "@patternfly/react-styles";
+import { uniqueId } from "utils";
+
+const styles = StyleSheet.create({
+  data_list_cell: {
```

Comment:
This style does not seem to being used anywhere

Reply:
done